### ISSUE TITLE
suggestion: escape other players?

### ISSUE THREAD

**Issue by Octolus** [davidmann4/ogar-feeder-bot#107]
i have a pretty good dedicated server, and i would think such feature would use some cpu.. for people like me, it would be pretty awesome so the bots would basically escape other users when going to me.

escape + eat even..

**Comment by davidmann4** [comment]
yeah its half done - currently they avoid green balls (virus) shoudl be easy to make it work with players

**Comment by Octolus** [comment]
that's awesome, can't wait to see it added
